scheduler/test/cook/test/mesos/api.clj

```diff
@@ -1208,6 +1208,7 @@
                    :retries_remaining 1
                    :state "waiting"
                    :status "waiting"
+                   :supports_data_locality false
```

Comment:
Should `supports_data_locality` only be displayed if the user explicitly specified it on the job?